hack/cloudbuild-release.yaml

```diff
@@ -16,7 +16,7 @@
 # as well as "latest"
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '-t', 'builder', '-f', 'hack/Dockerfile.build', '.']
+    args: ['build', '-t', 'builder', '-f', 'hack/Dockerfile', '.']
```

Comment:
this seems like an unrelated change?

Reply:
I explained in the commit message: if the Dockerfile has this non-standard name, it is not picked up by the license boilerplate validation.